### ISSUE TITLE
Missing files for LePetitPrince

### ISSUE THREAD

**Issue by ferponcem** [individual-brain-charting/public_analysis_code#66]
In drago, for all subjects, we have only some of the fsaverage5 and fsaverage7 derivatives.
Specifically, we only have the *effect_size_maps* for the ffx effects.

The stat_maps, the ap and pa derivatives and the volumetric maps are missing, see example in `derivatives/sub-04/ses-27`

**Comment by bthirion** [comment]
Hm, it's typically a case where the derivatives are not well defined (like movie watching). Not sure  what to do about that. But let's keep the issue open for now.

**Comment by ferponcem** [comment]
Yes! This is the same case for *LePetitPrice*, *GoodBadUgly* and *MonkeyKingdom*. 
To be discussed 👀
